packages/preview/clean-cnam-template/1.0.0/README.md

```diff
@@ -0,0 +1,149 @@
+# CNAM TYPST Template
+
+A modular and organized template for creating professional documents using CNAM branding and styling.
+
+Originally based on [hzkonor's bubble-template](https://github.com/hzkonor/bubble-template) and uses [CNAM](https://www.cnam.fr/)'s logo and colors.
+
+## Project Structure
+
+```
+├── lib/               # Template library files
+│   ├── config.typ     # Main configuration and document setup
+│   ├── fonts.typ      # Centralized font configuration
+│   ├── components.typ # UI components (blockquote, my-block, code)
+│   ├── headers.typ    # Header management logic
+│   ├── layout.typ     # Document layout and styling
+│   ├── utils.typ      # Utility functions
+│   ├── colors.typ     # Color definitions
+│   └── math.typ       # Mathematical environments
+├── assets/            # Static assets (logos, images, fonts)
+├── main.typ           # Main document file
+├── lib.typ            # Main package entrypoint
+└── README.md
+```
+
+## Features
+
+- **Modular Design**: Template split into logical, maintainable modules
+- **CNAM Branding**: Official CNAM colors and styling
+- **Centralized Font Configuration**: Global font settings for consistency
+- **Enhanced Code Blocks**: Syntax highlighting, line numbers, filename labels
+- **Rich Components**: Custom blockquotes, styled content blocks
+- **Math Support**: Mathematical definitions, theorems, examples
+- **Smart Headers**: Context-aware page headers
+- **Responsive Layout**: Professional document layout with decorative elements
+
+## Usage
+
+1. Import the template in your document:
+```typst
+#import "@preview/clean-cnam-template:1.0.0": *
+```
+
+2. Configure your document:
+```typst
+#show: conf.with(
```

Comment:
This doesn't compile because it should be `cnam-typst-template` instead of `conf`. But honestly I would consider renaming the `cnam-typst-template` function to either the package name or removing the `typst`.
I think the changelog also still has the old name listed under package information.